fxa-client/ffi/src/lib.rs

```diff
@@ -202,6 +207,22 @@ pub extern "C" fn fxa_get_token_server_endpoint_url(
     })
 }
 
+/// Get the url to redirect after there has been a successful connection to FxA.
+///
+/// # Safety
+///
+/// A destructor [fxa_str_free] is provided for releasing the memory for this
+/// pointer type.
+#[no_mangle]
+pub extern "C" fn fxa_get_connection_success_url(
+    fxa: &FirefoxAccount,
+    error: &mut ExternError,
+) -> *mut c_char {
+    call_with_result(error, || {
+        fxa.get_connection_success_endpoint_url().map(|u| u.to_string())
```

Comment:
Nice! One of the goodness about @thomcc changes is that we don't have to convert to `c_char` explicitely